docs/security.md

```diff
@@ -15,6 +15,24 @@
 
 # Security
 
+## Overview
+Trino-gateway has it's own security with it's own authetication and authorization.
+It's security is used only to authenticate/authorize it's own ui and the APIs.
```

Comment:
Both of the It's should have the apostrophe removed, it's is a contraction whereas its is possessive.